build_tools/benchmarks/generate_benchmark_comment.py

```diff
@@ -344,56 +190,117 @@ def parse_arguments():
       nargs="+",
       help=("Paths to the JSON files containing compilation statistics, "
             "accepts wildcards"))
-  parser.add_argument("--dry-run",
-                      action="store_true",
-                      help="Print the comment instead of posting to GitHub")
+  parser.add_argument("--pr_number", required=True, type=int, help="PR number")
+  parser.add_argument("--pr_commit",
+                      required=True,
+                      type=str,
+                      help="PR commit hash")
   parser.add_argument(
-      "--query-base",
-      action="store_true",
-      help=("Query the dashboard for the benchmark results of the targeting "
-            "base branch"))
-  parser.add_argument("--comment-title",
-                      default=ABBR_PR_COMMENT_TITLE,
+      "--pr_base_commit",
+      type=str,
+      default=None,
+      help="Start commit to find the benchmark results to compare in reverse, "
+      "requires repo to have the full ancestor history of the start commit")
+  parser.add_argument("--comment_title",
+                      default="Abbreviated Benchmark Summary",
                       help="Title of the comment")
+  parser.add_argument(
+      "--comment_type_id",
+      default="f6919a4c-7bb3-4fd6-af89-97980ce49f95",
```

Comment:
It's an id to help find the same kind of comment in PR and update it. I think it's better to be a random id so we can freely change the comment title without worrying about being irrelevant to the id and avoid creating a new comment because of the change of title. 

I changed it to a readable id to help understand the code. But in CI I still assign a random id on it.

Reply:
I still don't follow why this arg would this have a default at all though. I like having artificial keys, but I don't think users should have to be passing them around. What about we have the user pass a human-readable thing that maps to a uuid stored in the code. Then we have a stable artificial key that is what we actually put in the PR comments, but users of the CLI (including the CI) pass a human-readable identification. We then map that to the UUID. We can freely change the user-facing ID since it's only stored in code.

Reply:
I see. SG

I decided to just have a `--comment_type` with the predefined human-readable options. The option is then mapped to comment title and id.